app/views/about/index.html.erb

```diff
@@ -13,6 +13,7 @@
           <li><a href="#compute_resources" data-toggle="tab"><%= _('Compute Resources') %></a></li>
         <% end %>
         <li><a href="#plugins" data-toggle="tab"><%= _('Plugins') %></a></li>
+        <li><a href="#safemode" data-toggle="tab"><%= _('Safemode') %></a></li>
```

Comment:
ditto about "Safemode", consider making this more generic - "Template help", "Help" or similar?